easybuild/easyconfigs/n/NCCL/NCCL-2.16.2-GCCcore-12.2.0-CUDA-12.0.0.eb

```diff
@@ -0,0 +1,43 @@
+easyblock = 'Tarball'
```

Comment:
@robogast The faulty checksum for `aarch64` reminds me: why are you using pre-built binaries here, instead of installing NCCL from source?

Reply:
NCCL v2.16.2 is not available in its GitHub repo...

Reply:
It seems they made the commit, but forgot to tag it: https://github.com/NVIDIA/nccl/commit/28189e2df85885b78bf20384cefc3cedab023bf5

Reply:
https://github.com/NVIDIA/nccl/issues/755 created an issue for it.